packages/fuels-contract/src/contract.rs

```diff
@@ -379,20 +379,18 @@ impl Contract {
             })
     }
 
-    /// Deploys a compiled contract to a running node
```

Comment:
Don't remove the docstring

Reply:
Was this actually resolved? I'm still seeing the docstring removed.

Reply:
My bad